### DISCUSSION TITLE
DEXATTACH - Unexplained error message

### DISCUSSION THREAD

**Discussion by mmongy** [retronas/discussions#114] [Q&A]
This error message:

xxx@xxx-Asus-P8Z77-V-Pro:~$ sudo dexattach --check --verbose /dev/ttyUSB0
Opening /dev/ttyUSB0 and setting line discipline
Cannot set line discipline: No such device or address

is not explained at all. I can't figure how to fix the problem.

Sincerely, Marc.

**Comment by mmongy** [comment]
The dexdrive is plugged, before booting the computer.

**Comment by sairuk** [comment]
We don't write this tool, we provide the installer.

Is your usb serial port actually at `/dev/ttyUSB0`, the message suggests that is not the case.

I see you have already raised an issue on the main project here https://github.com/fbriere/linux-dexdrive/issues/2

**Comment by mmongy** [comment]
Hi @sairuk,

Thanks for your response, even though you are not in charge of this project. In fact, i noticed i had posted my request to the bad person, but too late.

However, there is a silver lining. With my Dexdrive, the AC/DC converter was not made for the local alternative current supply, so i relied on a multipurpose power supply (multiple voltages, multiple DC outlet shapes).

After rechecking, i noticed the DC outlet was a bad one. The external diameter (negative current) of the plug was OK, but the internal diameter (positive current) was too large and didn't made any contact with the central pin of the DC inlet of my device.

It's as dumb as this. Sorry for the inconvenience. Now i have this message:

xxx@xxx-Asus-P8Z77-V-Pro:~$ sudo dexattach --check --verbose /dev/ttyUSB0               
Opening /dev/ttyUSB0 and setting line discipline
Cannot set line discipline: Invalid argument

Some tunings are still needed, but my device is properly power supplied now. 

Thanks anyway, Marc.

**Comment by sairuk** [comment]
ok good luck I will convert this to a discussion for now
